Memcached: fix DSM 6 installation procedure

smaarn opened the pull request:
## Description

Fixing regression introduced on DSM 6 installation procedure where the web package part wasn't installed (nor uninstalled for what it's worth).

Fixes https://github.com/SynoCommunity/spksrc/issues/5343

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)


### Type of change

- [x] Bug fix

smaarn commented:
Binaries available here: https://github.com/smaarn/spksrc/releases/tag/memcache-1.6.15-4-rc-1

hgy59 commented:
@smaarn I am confused, as I tested version 1.6.15-3 on my DS-218 with DSM 6.2.4-25556. 
Just uninstalled memcached now and reinstalled again and it still runs without any issue.

smaarn commented:
The uninstaller and installer were not touching the web services directory. My guess is that after uninstalling the package the web phpmemcached directory is left untouched?

smaarn commented:
New binaries for testing here: https://github.com/smaarn/spksrc/releases/tag/memcache-1.6.15-4-rc-2

smaarn commented:
@hgy59 I thought I had replied to you but it seems that I had forgotten to tag you.

I think that the issue was the fact that uninstallation was not removing anything either so reinstallation was working fine. I believe the issue occurs when upgrading an older version or installing from scratch which wouldn't work.

But then I may have missed something obviously.

smaarn commented:
Rebased at head and squashed commits to make explicit the full changes and avoid any eventual back and forths ones in preparation of an eventual merge.
